tests/baselines/reference/symbolProperty49.errors.txt

```diff
@@ -0,0 +1,13 @@
+tests/cases/conformance/es6/Symbols/symbolProperty49.ts(5,10): error TS2471: A computed property name of the form 'Symbol.iterator' must be of type 'symbol'.
+
+
+==== tests/cases/conformance/es6/Symbols/symbolProperty49.ts (1 errors) ====
+    module M {
+        export var Symbol;
+    
+        class C {
+            [Symbol.iterator]() { }
+             ~~~~~~~~~~~~~~~
+!!! error TS2471: A computed property name of the form 'Symbol.iterator' must be of type 'symbol'.

```

Comment:
No, the type of Symbol.iterator here is `any`. Although `any` is normally allowed in computed properties, when you write it in this form (Symbol.something), it is a pretty strong indication that you mean to use a symbol, so we give you an error because you are probably using symbols wrong.